jsbi.mjs

```diff
@@ -951,7 +951,7 @@ class JSBI extends Array {
   }
 
   static __compareToNumber(x, y) {
-    if (y | 0 === 0) {
+    if (isInt(y)) {
```

Comment:
Is this change actually correct? The original code followed the format `(arg | 0) === 0`, not the format `(arg | 0) === arg` which `isInt` implements.

Reply:
```
(100 | 0) === 0 // false
(100 | 0) === 100 // true
```

Yes - this code was originally buggy, and was rejecting valid integers.